include/swift/AST/Decl.h

```diff
@@ -2319,7 +2326,8 @@ class ValueDecl : public Decl {
   /// A public declaration is accessible everywhere.
   ///
   /// If \p DC is null, returns true only if this declaration is public.
-  bool isAccessibleFrom(const DeclContext *DC) const;
+  bool isAccessibleFrom(const DeclContext *DC,
+                        bool forConformance=false) const;
```

Comment:
Please include an explanation of the `forConformance` flag in the doc comment. This whole thing was really subtle and deserves to be written down.

Reply:
Done